dbld/packages.manifest

```diff
@@ -154,14 +154,13 @@ lcov                            [devshell]
 libc6-dbg                       [devshell]
 libglib2.0-0-dbgsym             [devshell]
 libjemalloc-dev                 [devshell]
-libjemalloc1-dbgsym             [devshell]
-libpcre2-dbg                    [devshell]
-libssl1.0.0-dbg                 [devshell]
+libjemalloc2-dbgsym             [devshell]
+libssl1.1-dbgsym                [devshell]
 linux-tools-generic             [devshell]
 locales                         [devshell]
 lsof                            [devshell]
 netcat                          [devshell]
-python-dbg                      [devshell]
+python2-dbg                     [devshell]
```

Comment:
Don't we need `python2-dev` packages, too?

Reply:
that's probably taken care of by build dependencies.

Reply:
I don't think it's taken care of:
On this branch:
```
$ ./dbld/rules image-devshell
$ docker run -it balabit/syslog-ng-devshell:latest bash
root@4a82e33401a6:/# apt search python2-dev
Sorting... Done
Full Text Search... Done
games-python2-dev/focal 2.2ubuntu3 all
  development of games in Python 2

libpython2-dev/focal 2.7.17-2ubuntu4 amd64
  header files and a static library for Python2

python2-dev/focal 2.7.17-2ubuntu4 amd64
  header files and a static library for Python2

root@4a82e33401a6:/# apt search python3-dev
Sorting... Done
Full Text Search... Done
games-python3-dev/focal 2.2ubuntu3 all
  development of games in Python 3

libpython3-dev/focal,now 3.8.2-0ubuntu2 amd64 [installed,automatic]
  header files and a static library for Python (default)

python3-dev/focal,now 3.8.2-0ubuntu2 amd64 [installed]
  header files and a static library for Python (default)

python3-devpi-common/focal 3.2.2-1 all
  PyPI server and packaging/testing/release tool - Common modules
```

No `[installed]'.

Reply:
Oh, I see, it might be taken care of when packaging syslog-ng.

My fear is, that someone pulls the devshell image from dockerhub, clones the source code, and tries to build syslog-ng, and cannot because of the missing of `python2` packages.

Reply:
build deps should be installed in the base image. But I see your point, the devshell image should have both python2 and python3 regardless if we build syslog-ng with python3 on focal by default. With this in mind we would really need to specify python2-dev explicitly.

Reply:
added python-dev to packages.manifest